src/components/ha-code-editor.ts

```diff
@@ -172,7 +171,7 @@ export class HaCodeEditor extends ReactiveElement {
     });
   }
 
-  private _getStates = memoizeOne((states: HassEntities | null | undefined) => {
```

Comment:
Actually, the way I did it now `memoizeOne` does make sense, as it rebuilds the options every time the autocomplete is opened, instead of just on creation of the editor.

Reply:
Damn, I was too quick 😅

Reply:
Will revert that

Reply:
Already did it for you :-)

Reply:
Awesome, thanks!